python/samples/getting_started_with_agents/step7_assistant.py

```diff
@@ -59,7 +59,7 @@ async def main():
     kernel = Kernel()
 
     # Add the sample plugin to the kernel
-    kernel.add_plugin(plugin=MenuPlugin(), plugin_name="menu")
+    kernel.add_plugin(plugin=MenuPlugin(), name="menu")
```

Comment:
Is the name of the parameter supposed to be `name` or `plugin_name`? I believe it's `plugin_name`, no?